src/app/routes/issues.new.tsx

```diff
@@ -103,6 +103,25 @@ export async function action({ request }: ActionFunctionArgs) {
                         )
                         .catch(console.error);
 
+                    await sendDiscordWebhookWithUrl(config.discord.studentServerIssueWebhookUrl, {
+                        thread_name: `${data.title} - #${issue.id}`,
```

Comment:
Switch id and title - when there are many issues with titles of variable lenghts it looks a bit nicer if the issue # is in front. I noticed quite a few people talk about the issue number when referring to a specific issue, not the title.
```suggestion
                        thread_name: `#${issue.id} - ${data.title}`,
```